pss/outbox/outbox_whitebox_test.go

```diff
@@ -72,3 +79,21 @@ func TestFullOutbox(t *testing.T) {
 		t.Fatalf("timeout waiting for a free slot")
 	}
 }
+
+const blockTimeout = 100 * time.Millisecond
+
+func expectNotTimeout(completionC chan struct{}, t *testing.T) {
```

Comment:
I think test helpers generally have `t *testing.T` as a first argument, not as a last. I find this construct a bit weird when `t` is the last argument.